packages/web-twig/src/Resources/components/FileUploader/README.md

```diff
@@ -0,0 +1,259 @@
+
+FileUploader is a composition of a few subcomponents:
+
+- [FileUploader](#fileuploader-1)
+  - [FileUploaderInput](#fileuploaderinput)
+  - [FileUploaderList](#fileuploaderlist)
+    - [FileUploaderAttachment](#fileuploaderattachment)
+
+## JavaScript Plugin
+
+For full functionality, you need to provide Spirit JavaScript:
+
+```html
+<script src="node_modules/@lmc-eu/spirit-web/js/cjs/spirit-web.min.js" async></script>
+```
+
+Check the [component's docs in the web package][web-js-api] to see the full documentation of the plugin.
+
+Please consult the [main README][web-readme] for how to include JavaScript plugins.
+
+Or, feel free to write the controlling script yourself.
+
+## FileUploader
+
+This is the main wrapper for the whole composition. It provides proper spacing for its subcomponents:
+
+```twig
+<FileUploader data-toggle="fileUploader">
+  <!-- FileUploaderInput -->
+  <!-- FileUploaderList -->
+</FileUploader>
+```
+
+### Fluid Width
+
+By adding the `isFluid` attribute, FileUploader can take up all the available horizontal space:
+
+```twig
+<FileUploader isFluid data-toggle="fileUploader">
+  <!-- FileUploaderInput -->
+  <!-- FileUploaderList -->
+</div>
+```
+
+### Maximum File Size (JavaScript)
+
+The maximum size of the uploaded file that is validated by the JavaScript plugin can be adjusted. The default value is
+10 MB. To increase the limit for example to 20 MB, add the `data-max-file-size` attribute:
+
+```twig
+<FileUploader data-toggle="fileUploader" data-max-file-size="20000000">
+  <!-- FileUploaderInput -->
+  <!-- FileUploaderList -->
+</FileUploader>
+```
+
+### API
+
+| Prop name | Type      | Default | Required | Description                                                |
+| --------- | --------- | ------- | -------- | ---------------------------------------------------------- |
+| `isFluid` | `boolean` | `false` | no       | If true, the element spans to the full width of its parent |
+
+On top of the API options, you can add `data-*` or `aria-*` attributes to
+further extend component's descriptiveness and accessibility. Also, UNSAFE styling props are available,
+see the [Escape hatches][escape-hatches] section in README to learn how and when to use them.
+These attributes will be passed to the topmost HTML element of the component.
+
+## FileUploaderInput
+
+FileUploaderInput is a file picker built around the native HTML [file `input`][mdn-input-file].
+
+If supported by the device, FileUploaderInput automatically turns on the drag-and-drop functionality.
+
+```twig
+<FileUploaderInput
+    helperText="Max file size is 10 MB"
+    id="example-input"
+    label="Label"
+    name="example-input"
+/>
+```
+
+### Uploading Multiple Files
+
+To pick more than one file, just add the [`multiple`][mdn-multiple] attribute that will be transferred to the native
+HTML input:
+
+```twig
+<FileUploaderInput
+  id="example-input"
+  label="Label"
+  multiple
```

Comment:
```suggestion
  isMultiple
```

Reply:
?

Reply:
Along with `accept`, `multiple` is listed in the allowed attributes that are transferred to the native input:

```twig
{%- set _allowedAttributes = [ 'accept', 'multiple' ] -%}
```

![obrazek](https://github.com/lmc-eu/spirit-design-system/assets/5614085/2cb5b92c-de46-49df-807b-9afb4c6355e3)

I don't work with the attributes anywhere in the code. Or do you think they should be part of the API?

Reply:
As agreed on Slack, we are leaving it as it is. We may change it later when the API review against React implementation (#804) is done.